src/main/java/com/google/sps/EventFormInput.java

```diff
@@ -0,0 +1,59 @@
+package com.google.sps;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
```

Comment:
We already have a servlet that handles the form to create a new event (address 
src/main/java/com/google/sps/servlets/CreateEvent.java)